src/librustc/mir/mod.rs

```diff
@@ -2769,6 +2769,13 @@ pub struct BorrowCheckResult<'tcx> {
     pub used_mut_upvars: SmallVec<[Field; 8]>,
 }
 
+/// The result of the `mir_const_qualif` query.
+#[derive(Clone, Copy, Debug, Default, RustcEncodable, RustcDecodable, HashStable)]
+pub struct QualifSet {
```

Comment:
I don't think "set" is relevant anymore, the bitset was really just an optimization.
`ConstQualifs` might be better. Or `ValueQualifs`?

Reply:
I would lean towards `ConstQualifs` since `IsNotPromotable` is no longer, but either is fine.